docs/content/reference.md

```diff
@@ -41,9 +41,9 @@ Listener functions have access to a set of arguments that may change based on th
 
 | Argument  | Listener | Description  |
 | :--- | :--- | :--- |
-| `payload` | All listeners | The unwrapped contents of the incoming event, which varies based on event. This is a subset of the information included in `body` which is detailed below. `payload` is also accessible via the alias corresponding to the method name that the listener is passed to (`message`, `event`, `action`, `shortcut`, `view`, `command`, `options`) **An easy way to understand what's in a payload is to log it**, or [use TypeScript](/tutorial/using-typescript). |
+| `payload` | All listeners | The unwrapped contents of the incoming event, which varies based on event. This is a subset of the information included in `body` which is detailed below. `payload` is also accessible via the alias corresponding to the method name that the listener is passed to (`message`, `event`, `action`, `shortcut`, `view`, `command`, `options`) **An easy way to understand what's in a payload is to log it**, or [use TypeScript](https://github.com/slackapi/bolt-js/tree/main/examples/getting-started-typescript). |
```

Comment:
This page was removed in 52687bffc9ab0482aa8a3d8cef31c944bdfccf86 but I believe the included example is alright for getting started here!